templates/model/spec/v1alpha1/upgrade.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"fmt"
+
+	"github.com/abcxyz/abc/templates/model"
+	"github.com/abcxyz/abc/templates/model/spec/v1beta1"
+	"github.com/jinzhu/copier"
+)
+
+func (s *Spec) Upgrade() (model.ValidatorUpgrader, error) {
+	out := &v1beta1.Spec{}
```

Comment:
Style nit:

```suggestion
	var out v1beta1.Spe
```

and then `&out`

Reply:
Done.